src/jvm/test/org/jetbrains/kotlinx/lincheck_test/util/InternalLincheckExceptionEmulator.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Lincheck
+ *
+ * Copyright (C) 2019 - 2023 JetBrains s.r.o.
+ *
+ * This Source Code Form is subject to the terms of the
+ * Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
+ * with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+@file:Suppress("PackageDirectoryMismatch")
+
+package org.jetbrains.kotlinx.lincheck.util
+
+/**
+ * Utility object to emulate exception in the Lincheck itself
```

Comment:
Please explain that the package name differs, and that is how Lincheck understands whether the code internal or not, blah-blah-blah.